### PR TITLE
Added 2 grid examples

### PR DESCRIPTION
**Pull request by piixelcat**
Added 2 grid examples

### CODE REVIEW

File: grid/p5js/corridor/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Julia Hilt
+author-url: https://github.com/piixelcat
+title: corridor
+tags: P5.js, grid, "Law of Space"
```
Comment (fabianmoronzirfas) [inline grid/p5js/corridor/index.md]: Tags need to be in an array e.g.:  

    tags: [P5.js, grid, "Law of Space",]

File: grid/p5js/corridor/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+permalink: /grid/corridor/
+summary: narrow corridor
```
Comment (fabianmoronzirfas) [inline grid/p5js/corridor/index.md]: Can please add at least a sentence.

File: grid/p5js/lines/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  colorMode(HSB);
+}
+
+function draw() {
+  background(98);
+  
+  for (var y = 0; y <= 500; y += 5){
+    stroke(30);
+    strokeWeight(2);
+    line(0, y, 500, y);
```
Comment (fabianmoronzirfas) [inline grid/p5js/lines/index.js]: Use width instead of magic number

File: grid/p5js/lines/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Julia Hilt
+author-url: https://github.com/piixelcat
+title: lines
+tags: P5.js, grid, "Law of Prägnanz"
```
Comment (fabianmoronzirfas) [inline grid/p5js/lines/index.md]: tags need to be in an array

File: grid/p5js/lines/index.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+permalink: /grid/lines/
+summary: many lines and one square
```
Comment (fabianmoronzirfas) [inline grid/p5js/lines/index.md]: Make it at least one sentence.

File: grid/p5js/corridor/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+function setup() {
+  createCanvas(500, 500);
+  colorMode(HSB);
+  rectMode(CENTER);
+  background(98);
+  
+  stroke(15);
+  line(width/2, 4*height/10, width/2, 6*height/10);
+  
+  for (var x = 0; x <=500; x +=25){
```
Comment (fabianmoronzirfas) [inline grid/p5js/corridor/index.js]: use the width instead of magic numbers

File: grid/p5js/lines/index.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+function setup() {
+  createCanvas(500, 500);
+  colorMode(HSB);
+}
+
+function draw() {
+  background(98);
+  
+  for (var y = 0; y <= 500; y += 5){
```
Comment (fabianmoronzirfas) [inline grid/p5js/lines/index.js]: this is also a magic number for the width

**Comment by piixelcat** [comment]
Did all changes as requested.

**Comment by fabianmoronzirfas** [comment]
Awesome! Thank you. I will push the changes to the site ASAP (something like next week).

**Pull request merged**